repository.yaml

```diff
@@ -2,4 +2,6 @@ templates:
   - name: TypeScript - Starter
     path: ./typescript-starter
   - name: JavaScript - Starter
-    path: ./javascript-starter
\ No newline at end of file
+    path: ./javascript-starter
+  - name: Golang - Starter
```

Comment:
we should also flag the support state in the template name for golang, to make it clear that that dedicated documentation and support aren't available at this stage (Jye suggested flagging it with community for now)

Reply:
we will need support for that in the cli first